tests/testutils.py

```diff
@@ -70,8 +70,8 @@ def subprocess_run(*args, **kwargs):  # type: (str, Any) -> subprocess.Completed
 
 
 def validate_wheel_contents(
-    name, version, path, files=None
-):  # type: (str, str, str, Optional[List[str]]) -> None
+    name: str, version: str, path: str, files: Optional[List[str]] = None
+) -> None:
     dist_info = "{}-{}.dist-info".format(name, version)
     files = files or []
```

Comment:
does the following work?

```suggestion
    name: str, version: str, path: str, files: [List[str] = []
) -> None:
    dist_info = "{}-{}.dist-info".format(name, version)
```

Reply:
[Python Mutable Defaults Are The Source of All Evil](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/) :smiley:

Reply:
> [Python Mutable Defaults Are The Source of All Evil](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/) smiley

ah. yep.